release/pkg/test/testdata/main-bundle-release.yaml

```diff
@@ -1166,6 +1175,15 @@ spec:
         uri: https://release-bucket/artifacts/v0.0.0-dev-build.0/kind/manifests/kindnetd/v0.14.0/kindnetd.yaml
       version: v0.14.0+abcdef1
     kubeVersion: "1.21"
+    nutanix:
+      clusterAPIController: {}
+      clusterTemplate:
+        uri: https://release-bucket/artifacts/v0.0.0-dev-build.0/cluster-api-provider-nutanix/manifests/infrastructure-nutanix/v0.5.1/cluster-template.yaml
+      components:
+        uri: https://release-bucket/artifacts/v0.0.0-dev-build.0/cluster-api-provider-nutanix/manifests/infrastructure-nutanix/v0.5.1/infrastructure-components.yaml
+      metadata:
+        uri: https://release-bucket/artifacts/v0.0.0-dev-build.0/cluster-api-provider-nutanix/manifests/infrastructure-nutanix/v0.5.1/metadata.yaml
+      version: v0.5.1+abcdef1
```

Comment:
@thunderboltsid where is +abcdedf1 coming from?

Reply:
🤷‍♂️

Reply:
It's an autogenerated file.

Reply:
It comes from constants.go, it's a dummy value. We want to make the testdata file deterministic and reproducible.